content/en/contribute/medic/onboarding/technical-resources.md

```diff
@@ -0,0 +1,37 @@
+
+## Technical Resources
+
+| Skill | Resources |
+| ----------------| ----------- |
+| **CHT Local Environment and Docs** | [Set up your local development environment]({{% ref "dev-environment" %}}) <br> [Go through CHT Documentation]({{% ref "/" %}}) <br> [Get familiar with our Development Workflow]({{% ref "/contribute/code/workflow" %}}) <br>[Architecture of CHT Instances]({{% ref "architecture" %}}) <br> <br> Internal resources: <br> [App building for techies crash course with Ashley Jones](https://drive.google.com/drive/folders/1PTe8RH59TPBNYKoKzlZ_ZwMlQedKRGlx) <br>  [Where to ask questions about the CHT?](https://www.notion.so/medicmobile/CHT-Forum-Internal-Guide-c2d1988a116244b6b17b3aea284ff8ee) |
+| **JavaScript** | [Eloquent JavaScript](https://eloquentjavascript.net/) <br> [Asynchronous JavaScript Requests](https://www.udacity.com/course/asynchronous-javascript-requests--ud109) <br> [You Don't Know JS](https://github.com/getify/You-Dont-Know-JS) |
+| **TypeScript** | [TypeScript Documentation](https://www.typescriptlang.org/docs/) |
+| **Docker** | [Containers and Docker Mega Thread](https://twitter.com/iximiuz/status/1423984739514454033?s=21) <br> [Docker mastery course](https://www.udemy.com/course/docker-mastery/) <br> [Learn Docker in a month of lunches](https://diamol.net/) <br> [The Ultimate Docker course](https://codewithmosh.com/p/the-ultimate-docker-course) |
+| **CouchDB and PouchDB** | [CouchDB 2.3 introduction user guide docs](https://docs.couchdb.org/en/stable/intro/index.html) <br> [Understanding CouchDB 3](https://www.udemy.com/course/understanding-couchdb/) *_This is not the same version used in the CHT_ <br> [Introduction to PouchDB](https://pouchdb.com/guides/) |
+| **NodeJS and NPM** | [The Complete Node.JS developer course](https://www.udemy.com/course/the-complete-nodejs-developer-course-2/) <br> [Understanding NPM - Node.js Package Manager](https://www.udemy.com/course/understanding-npm/)|
+| **Angular** | [Angular 12 - Complete Beginner's Guide 2021](https://www.udemy.com/course/learning-angular/)|
+| **Offline First** | [Offline Web Applications](https://www.youtube.com/playlist?list=PLAwxTw4SYaPmTSxtOWyJVKTUaNBGze2ed) |
+| **Web Architecture** | [Web Architecture 101](https://medium.com/storyblocks-engineering/web-architecture-101-a3224e126947) |
+| **Web Browsers** | [Inside look at modern web browsers, part I](https://developer.chrome.com/blog/inside-browser-part1/) <br> [Inside look at modern web browsers, part II](https://developer.chrome.com/blog/inside-browser-part2/) <br> [Inside look at modern web browsers, part III](https://developer.chrome.com/blog/inside-browser-part3/) <br> [Inside look at modern web browsers, part IV](https://developer.chrome.com/blog/inside-browser-part4/)|
+| **Web Performance** | [High Performance Browser Networking](https://hpbn.co/)|
```

Comment:
I expect this table will grow over time. Would it be more accessible to use headers and lists instead of table format?

eg:

### CHT Local Environment and Docs

- [Set up your local development environment]({{% ref "dev-environment" %}})
- [Go through CHT Documentation]({{% ref "/" %}}) <br> [Get familiar with our Development Workflow]({{% ref "/contribute/code/workflow" %}})

... etc

Reply:
I prefer the table style for the readability and conciseness of the page. Yes, we might add links in the future, but it will only happen sometimes. Let's keep it this way for now and change the style if we get to a point where it is not readable anymore.